UPGRADING.rst

```diff
@@ -10,6 +10,40 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2399 Reduce portal DB IT concurrency
+=====================================
+
+Delete the existing portal databases and their corresponding version tracking
```

Comment:
I think there should be a script for this. It's too involved.

Reply:
The script overwrites the database with its default value (hardcoded in the plugin) rather than deleting it, since this requires fewer changes to the `PortalService` class.